src/components/Nav/Nav.js

```diff
@@ -1,7 +1,142 @@
-import React from "react";
+import { React, useState } from "react";
+import styled from "styled-components";
 
 const Nav = () => {
-  return <div>네비게이션</div>;
+  const [borderA, setIBorderA] = useState(true);
+  const [borderB, setIBorderB] = useState(true);
```

Comment:
```suggestion
  const [currentHover, setCurrentHover] = useState("");
```
수정님 해당 동작은 하나의 state로 관리할 수 있는 기능입니다!
currentHover라는 state에는 hover 한 버튼이 left 인지 center인지 right 인지에 따라서
스타일을 다르게 줄 수 있겠습니다!